### PR TITLE
clusterlogging pkg: IsReady method added to the clusterlogging handler

### PR DESCRIPTION
**Pull request by elenagerman**

### CODE REVIEW

**Comment by elenagerman** [comment]
@trewest @kononovn @achuzhoy @cdvultur PTAL

File: pkg/clusterlogging/clusterlogging.go [hunk 1]
```diff
@@ -279,6 +283,32 @@ func (builder *Builder) WithVisualization(
 	return builder
 }
 
+// IsReady check if the clusterLogging instance state is Ready.
```
Comment (achuzhoy) [inline pkg/clusterlogging/clusterlogging.go:286]:
```suggestion
// IsReady checks for the duration of timeout if the clusterLogging instance state is Ready.
```

**Review by kononovn — APPROVED** [review]
/lgtm

**Review by kononovn — CHANGES_REQUESTED** [review]
Looks good. Please add test case for this function

**Review by kononovn — APPROVED** [review]
/lgtm

**Pull request merged**